packages/ui/IconButton/IconButton.js

```diff
@@ -156,11 +157,17 @@ const IconButtonBase = kind({
 			children = null;
 		}
 
+		console.log('css:', css);
 		return (
-			<Button {...rest} small={small} minWidth={false}>
+			<ComponentOverride
```

Comment:
We need to update the prop type to support objects as well now that we've added `ComponentOverride`.